experiments/ClimaEarth/components/land/climaland_bucket.jl

```diff
@@ -323,7 +322,7 @@ function FluxCalculator.surface_thermo_state(
     # Note that the surface air density, ρ_sfc, is computed using the atmospheric state at the first level and making ideal gas
     # and hydrostatic balance assumptions. The land model does not compute the surface air density so this is
     # a reasonable stand-in.
-    ρ_sfc = Interfacer.get_field(sim, Val(:air_density))
+    ρ_sfc = extrapolate_ρ_to_sfc.(thermo_params, thermo_state_int, T_sfc) # ideally the # calculate elsewhere, here just getter...
```

Comment:
It might be cleaner to have the air density stored in the `sim`, but then we have to allocate it and carry it around just to access it this once, in a location where we also have access to everything we need to compute it.